### PR TITLE
Show warning message for expired users

### PR DESCRIPTION
**Pull request by zzarcon**
@sluceno :palm_tree:

### CODE REVIEW

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,5 +1,11 @@
-{{#if session.isLogged}}
-  {{partial "partials/logged"}}
+{{#if session.isExpired}}
+  <div class="session-error">
```
Comment (sluceno) [inline app/templates/application.hbs]: We could have this in a partial or sth, isn't it?

File: app/styles/sections/login.scss [hunk 2]
```diff
@@ -46,7 +46,12 @@
     @include transform(scale(0.6));
   }
 }
-
+.session-error{
```
Comment (sluceno) [inline app/styles/sections/login.scss]: Should we have this is a spscific file as well?

File: app/initializers/session.js [hunk 3]
```diff
@@ -54,7 +56,13 @@ export default {
...
 
       app.advanceReadiness();
-    }).fail(function() {
+    }).fail(function(response) {
+      var json = response.responseJSON;
+
+      if (json.error) {
+        session.set('error', json.error);
+      }
+
```
Comment (sluceno) [inline app/initializers/session.js]: This is to set the error in the Session model, isn't it?

File: app/styles/sections/login.scss (hunk 2 above)
Comment (zzarcon) [inline app/styles/sections/login.scss]: Yes, good idea

File: app/templates/application.hbs (hunk 1 above)
Comment (zzarcon) [inline app/templates/application.hbs]: mmm for me it's to much, creating another partial only for show a warning... but, in the end it's an opinion, you think that should be moved? try to convince me ;)

File: app/initializers/session.js (hunk 3 above)
Comment (zzarcon) [inline app/initializers/session.js]: Exactly

**Comment by zzarcon** [comment]
@sluceno I did the changes you suggest

**Pull request merged**